static/app/components/events/interfaces/nativeFrame.tsx

```diff
@@ -218,68 +218,54 @@ function NativeFrame({
   const status = getStatus();
 
   return (
-    <GridRow
-      inApp={frame.inApp}
-      expandable={expandable}
-      expanded={expanded}
-      className="frame"
-      data-test-id="stack-trace-frame"
-    >
+    <li data-test-id="stack-trace-frame">
       <StrictClick onClick={handleToggleContext}>
-        <StrictClickContent>
-          <StatusCell>
-            {(status === 'error' || status === undefined) &&
-              (packageClickable ? (
-                <PackageStatusButton
-                  onClick={handleGoToImagesLoaded}
-                  title={t('Go to images loaded')}
-                  aria-label={t('Go to images loaded')}
-                  icon={
-                    status === 'error' ? (
-                      <IconQuestion size="sm" color="errorText" />
-                    ) : (
-                      <IconWarning size="sm" color="errorText" />
-                    )
-                  }
-                  size="zero"
-                  borderless
-                />
-              ) : status === 'error' ? (
-                <IconQuestion size="sm" color="errorText" />
-              ) : (
-                <IconWarning size="sm" color="errorText" />
-              ))}
-          </StatusCell>
-          <PackageCell>
+        <RowHeader expandable={expandable} expanded={expanded}>
+          <div>
+            {status === 'error' ? (
+              <Tooltip
+                title={t(
+                  'This frame has missing deminification files and could not be symbolicated'
```

Comment:
Thanks for this great improvement, @doralchan 🥳.

On Mobile, it's called debug symbols. Deminification could be confusing to mobile engineers. Maybe you can change the wording.